src/algebra/algebra/subalgebra.lean

```diff
@@ -540,6 +540,11 @@ lemma mul_mem_sup {S T : subalgebra R A} {x y : A} (hx : x ∈ S) (hy : y ∈ T)
   x * y ∈ S ⊔ T :=
 (S ⊔ T).mul_mem (mem_sup_left hx) (mem_sup_right hy)
 
+lemma map_sup (f : A →ₐ[R] B) (S T : subalgebra R A) : (S ⊔ T).map f = S.map f ⊔ T.map f :=
```

Comment:
I assume it's too early to prove this via the map/comap Galois connection?

Reply:
Oh, I hadn't thought of that! I've golfed the proof using the Galois connection.